zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -57,8 +58,7 @@ def slack_workspace_to_realm(REALM_ID: int, user_list: List[ZerverFieldsT],
     3. added_recipient, which is a dictionary to map from channel name to zulip recipient_id
     4. added_channels, which is a dictionary to map from channel name to zulip stream_id
     """
-    # TODO fetch realm config from zulip config
-    DOMAIN_NAME = "zulipchat.com"
+    DOMAIN_NAME = settings.EXTERNAL_HOST
```

Comment:
This probably should be an option ("subdomain"), with this value as the default.

Reply:
@timabbott We are taking the subdomain as a parameter from the user itself. https://github.com/zulip/zulip/blob/master/zerver/lib/slack_data_to_zulip_data.py#L612

Reply:
Yeah, so should `DOMAIN_NAME` be that parameter, not `settings.EXTERNAL_HOST`?